src/cluecode/finder.py

```diff
@@ -46,28 +46,44 @@ def logger_debug(*args):
 Optionally apply filters to pattern matches.
 """
 
+def common_filter(line):
+	"""
+	Looks for the pattern that are common to both emails and urls
+	"""
+	commons = ["@", ".", ":", "/"]
+	bool = False
+	for pattern in commons:
```

Comment:
what about: `return any(p in line for p in "@.:/")`

Reply:
thanks @pombredanne , that is great.
please review my second commit.